pkg/service/restore/index.go

```diff
@@ -71,6 +72,12 @@ func (w *tablesWorker) indexLocationWorkload(ctx context.Context, location Locat
 	if err != nil {
 		return LocationWorkload{}, errors.Wrap(err, "create remote dir workloads")
 	}
+	if w.target.Continue {
+		rawWorkload, err = w.filterPreviouslyRestoredSStables(rawWorkload)
```

Comment:
Maybe worth to log the rawWorkload ?
Without including name of every SSTable, but by just counting how many of SSTables are per RemoteDirWorkload and how much of the data ?